.github/workflows/pythonpackage.yml

```diff
@@ -7,7 +7,7 @@ env:
 jobs:
   test:
 
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
I'm guessing old actions weren't saved to see what the runtime was before

Reply:
Too old; the logs are all cleaned. Based on a quick search, 20.04 is the last one that supports py36, arch x64 though, so that's why I put it here.